### PR TITLE
fix: improve chart interactivity

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
WEB-423

### Before

https://user-images.githubusercontent.com/109113616/197632771-3f2a1064-f657-4b55-ac95-6316e273fa9d.mov

### After

https://user-images.githubusercontent.com/109113616/197632857-afd4d20d-5284-424c-8c80-154b25c3727b.mov

## Test Plan
- Check if user can easily compare multiple trials

This feature is already implemented

## Checklist
- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/src/components/UPlot/UPlotChart/tooltipsPlugin.ts [hunk 1]
```diff
@@ -66,11 +68,11 @@ export const tooltipsPlugin = ({ getXTooltipHeader, getXTooltipYLabels }: Props
 
     // right
     if (chartWidth && idxLeft + tooltipWidth >= chartWidth) {
-      return idxLeft - tooltipWidth;
+      return idxLeft - tooltipWidth - margin;
     }
 
     // left
-    return idxLeft;
+    return idxLeft + margin;
```
Comment (EmilyBonar) [inline webui/react/src/components/UPlot/UPlotChart/tooltipsPlugin.ts:75]: question: what is this doing?

**Review by EmilyBonar — APPROVED** [review]
LGTM

**Comment by trentwatt** [comment]
do you think you could maybe make it so that the currently hovered trial is bold in the tooltip?

as is, it is kinda hard to tell which trial is the one you are hovering.

**Comment by trentwatt** [comment]
maybe i am not doing things right though.

**Comment by keita-determined** [comment]
> do you think you could maybe make it so that the currently hovered trial is bold in the tooltip?
> 
> as is, it is kinda hard to tell which trial is the one you are hovering.
> 
> image

I tried to make it bold or add a border, but couldnt in easy way cuz these tooltips are not built-in. 
Let me check if I can find a straightforward way. If not, i'll merge it

**Comment by keita-determined** [comment]
i'll merge it cuz i couldnt find a good way.
If this becomes an issue, we can come back

**Pull request merged**